source/_patterns/02-organisms/components/highlight~one-item.json

```diff
@@ -0,0 +1,30 @@
+{
+  "heading": {
+    "heading": "Highlights"
+  },
+  "items": [
+    {
```

Comment:
@viktoria-mahmud I think it looks great with 1 and 2 items. Good enough anyway. We will be insisting that 3 highlights for the homepage are recommended. If ever this changes then we can revisit how they are appearing with fewer items than 3.

Reply:
> Approving if the small changes I have requested here are made. Great work @viktoria-mahmud

The changes are ready. Thank you!